airflow/contrib/hooks/gcp_automl_hook.py

```diff
@@ -0,0 +1,727 @@
+            project=project_id,
+            location=location,
+            dataset=dataset_id,
+            table_spec=table_spec,
+        )
+        result = client.list_column_specs(
+            parent=parent,
+            field_mask=field_mask,
+            filter_=filter_,
+            page_size=page_size,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+        return result
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def get_model(
+        self,
+        model_id: str,
+        location: str,
+        project_id: str = None,
+        retry: Retry = None,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+    ) -> Model:
+        """
+        Gets a AutoML model.
+
+        :param model_id: Name of the model.
+        :type model_id: str
+        :param project_id: ID of the Google Cloud project where model is located if None then
+            default project_id is used.
+        :type project_id: str
+        :param location: The location of the project.
+        :type location: str
+        :param retry: A retry object used to retry requests. If `None` is specified, requests will not be
+            retried.
+        :type retry: Optional[Retry]
+        :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+            `retry` is specified, the timeout applies to each individual attempt.
+        :type timeout: Optional[float]
+        :param metadata: Additional metadata that is provided to the method.
+        :type metadata: Optional[Sequence[Tuple[str, str]]]
+
+        :return: `google.cloud.automl_v1beta1.types.Model` instance.
+        """
+        assert project_id is not None
+        client = self.get_conn()
+        name = client.model_path(project=project_id, location=location, model=model_id)
+        result = client.get_model(
+            name=name, retry=retry, timeout=timeout, metadata=metadata
+        )
+        return result
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def delete_model(
+        self,
+        model_id: str,
+        location: str,
+        project_id: str = None,
+        retry: Retry = None,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+    ) -> Model:
+        """
+        Deletes a AutoML model.
+
+        :param model_id: Name of the model.
+        :type model_id: str
+        :param project_id: ID of the Google Cloud project where model is located if None then
+            default project_id is used.
+        :type project_id: str
+        :param location: The location of the project.
+        :type location: str
+        :param retry: A retry object used to retry requests. If `None` is specified, requests will not be
+            retried.
+        :type retry: Optional[Retry]
```

Comment:
Can we provide a full package path for `Retry`? Without context, it's hard to know what the `Retry` object is